src/test/scala/io/qbeast/docs/DocumentationTests.scala

```diff
@@ -100,7 +100,15 @@ class DocumentationTests extends QbeastIntegrationTestSpec {
         "The sample should be more or less 10%"
       import spark.implicits._
 
-      val files = query.select(input_file_name()).distinct().as[String].collect()
+      val files = query
+        .select(input_file_name())
+        .distinct()
+        .as[String]
+        .map { path =>
+          val index = path.lastIndexOf('#')
```

Comment:
Maybe I am wrong, but does this mean that `input_file_name()` that has been written by Qbeast always contains the "#" in the path? Or is it just when we read as Qbeast datasource?

Reply:
When we reead a multiblock file we need to specify the row ranges of the blocks we wat to read, so the ranges are encoded as URI fragment like `#0-10,100-213,1000-1234`. The test checks the correctness of the file, so itis necessary to erase the fragmenr from the path.

Reply:
Yes, I understand. But does that mean that, when reading the Table as Delta, we will encounter a path like `"/tmp/test/f1.parquet#0-10"`? 

Or do these values just appear when reading as Qbeast? (For your comment I believe this is the case, but just to be sure)

Reply:
In case of Delta format we should not see those fragments. They should appear only in case of Qbeast. Actually we have two callback abstractions: FileIndex and FileFormat. The first one returns the URIs of the files to read and the second nows how to read them, so in case of Qbeast we have two implementations OTreeIndex and QbeastFileFormat, the first one encodes the row ranges into file URIs and the second one knows how to decode them and apply to the parquet files. In case of Delta they use their own implementation unrelated to Qbeast.

Reply:
Thanks for the explanation 👍